main.py

```diff
@@ -92,7 +95,8 @@ def parse_speed_table(table) -> dict:
 
                 if speeds:
                     vehicle_type = column_names[col_idx]
-                    speeds_by_vehicle_type[vehicle_type] = speeds.split(",")
+                    # TODO: Use these groups in the next stage to build a set of proper restrictions
+                    speeds_by_vehicle_type[vehicle_type] = [match.group() for match in SPEED_REGEX.finditer(speeds)]
```

Comment:
So the `,` to separate the limits is now optional. Weird strings like would be accepted by the parser, for example:
```
80
only with special plaquette: 100
```

I think it would be better if the parser was more strict because weird syntax or stray characters on the wiki page may as well be actual mistakes that should be fixed instead of leniently overlooked.
Additionally, if the syntax is not strictly checked, things like the above example - limitations that are understood by humans but not by a parser - may creep into the wiki page over time and go undetected.

Reply:
That’s correct. I wasn’t sure how you wanted to handle this kind of thing. I’ll tighten it up.

Reply:
@westnordost thanks for the improvements! Let me know if this PR looks good to you and I'll merge it. Then we can get started on #3 ;)